packages/climate-ref-core/src/climate_ref_core/metric_values/typing.py

```diff
@@ -6,6 +6,27 @@
 Value = float | int
 
 
+class SeriesDefinition(BaseModel):
+    """
+    A definition of a 1-d array with an associated index and additional dimensions.
+    """
+
+    file_pattern: str
```

Comment:
Maybe this is not generic enough to go here, as it assumes that output is stored in a file